src/test/resources/test-app.conf

```diff
@@ -33,8 +33,10 @@ app {
     uri = ${?IAM_BASEURI}
     test-realm = "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/nexus-internal"
     test-realm = ${?TEST_REALM}
-    service-account-token = ${SERVICE_ACCOUNT_TOKEN}
-    test-user-token = ${TEST_USER_TOKEN}
+    service-account-token = "TO_OVERRIDE"
+    service-account-token = ${?SERVICE_ACCOUNT_TOKEN}
```

Comment:
we don't need those environment variables override anymore, since we are using JVM properties.
```scala
service-account-token = null
```

Reply:
Yes, I let it this for later.
For running the tests both on CI and on or our laptops, I don't know which one is easier, it may be worth to keep environment variables.